requirements/AUR_CBI_MDL-0030/work/timeservice/implem/default/C/src/timeservice-api.c

```diff
@@ -0,0 +1 @@
+/home/taste/SpaceCreator/AURORA-Validation/AURORA-Reference-Component-Set/TimeService/work/timeservice/implem/default/C/src/timeservice-api.c
```

Comment:
Do we need to link to individual files? Would there be a possibility to link to the entire TimeService implementation?

Reply:
We were trying to link a directory but it appears that (as Rafał said) it is impossible to compile this way so we had to link individual files